src/Sulu/Bundle/SecurityBundle/Admin/SecurityAdmin.php

```diff
@@ -180,7 +188,9 @@ public function configureViews(ViewCollection $viewCollection): void
             );
         }
 
-        if ($this->securityChecker->hasPermission(static::USER_SECURITY_CONTEXT, PermissionTypes::EDIT)) {
+        if ($viewCollection->has(ContactAdmin::CONTACT_EDIT_FORM_VIEW)
```

Comment:
IIRC this is the missing link, which would make it possible to avoid [creating the `$contactEditFormView`](https://github.com/sulu/sulu/blob/6154460/src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php#L95-L99) in a different way. Maybe we should get rid of that in the `ContactAdmin` now? Would also allow us to remove [this weird `else`](https://github.com/sulu/sulu/blob/6154460/src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php#L176-L178).